packages/viewer/src/components/scene-tree/lib/controller.ts

```diff
@@ -496,14 +515,24 @@ export class SceneTreeController {
     term: string,
     options: FilterTreeOptions = {}
   ): Promise<void> {
-    await this.ifConnectionHasJwt((jwt) => {
-      return this.requestUnary(jwt, (metadata, handler) => {
-        const req = new FilterRequest();
-        req.setFilter(term);
-        req.setFullTree((options.includeCollapsed ?? true) === true);
-        if (options.columns) req.setColumnsKeysList(options.columns);
-
-        this.client.filter(req, metadata, handler);
+    return this.ifConnectionHasJwt(async (jwt) => {
+      const res = await this.requestUnary<FilterResponse>(
+        jwt,
+        (metadata, handler) => {
+          const req = new FilterRequest();
+          req.setFilter(term);
+          req.setFullTree((options.includeCollapsed ?? true) === true);
+          if (options.columns) req.setColumnsKeysList(options.columns);
+
+          this.client.filter(req, metadata, handler);
+        }
+      );
+
+      const { numberOfResults } = res.toObject();
```

Comment:
Can we add a test for this? Otherwise, looks good to me.

Reply:
Good idea 👍